# Goal
geometry_msgs/Pose goal
---
# Result
bool success
---
# Feedback
float32 feedback